sale_project_fixed_price_task_completed_invoicing/models/sale_order_line.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.model
+    def create(self, vals):
```

Comment:
@angelmoya it works fine. This bit is only when you create a line in state 'sale' and not draft, although I'm not quite sure when this can happen. @leemannd  can you enlighten us?